syntax = "proto3";
service Get_Params_Client{
    rpc get_params_client(params_request) returns (params_response){}
}
message params_request{

}
message params_response{
    int64 N = 1;
    int64 M = 2;
}